src/app/map/map.component.html

```diff
@@ -0,0 +1,19 @@
+
+<div class="container">
+    <div style="width: 100%;"><h1>Corona Virus Outbreak Map</h1></div>
+    <div style="height: 100%;width: 100%;">
+    <ac-map> 
+        <button class="button" (click) = "mapMarkClicked()">mark places</button>
```

Comment:
add ngif markingFlag because the button should dissapear after you click it (because its no longer needed)